scripts/wad-to-image

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# wad-to-image - Convert a WAD file to an image file.
+# by Steven Elliott <selliott512@gmail.com>
+#
+# The WAD file is displayed in Yadex, which must be installed, and then
+# "import" is used to capture the image in the Yadex window.
```

Comment:
Yadex is an unmaintained decrepid program that doesn't even compile for me ;)

Perhaps Eureka (Yadex derivative) or omgifol can do the job better.